### PR TITLE
Added fire tour story

### PR DESCRIPTION
**Pull request by minniewong**
## Description

Tour story for July

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- [ ] I have added necessary documentation (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules (if applicable)

## Further comments

If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

@nasa-gibs/worldview

### CODE REVIEW

File: config/default/common/config/metadata/stories/fires_overview_2019/step001.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<p>NASA provides insights into fires and thermal anomalies occurring daily around the world. Fire information available in Worldview are from the MODIS instrument on board the Terra and Aqua satellites and the VIIRS instrument on board the joint NASA/NOAA Suomi-NPP satellite. Terra passes over the equator at approximately 10:30am (Day) and 10:30pm (Night) local time and Aqua and Suomi-NPP passes over the equator at approximately 1:30pm (Day) and 1:30am (Night) local time. The fire information are available within Worldview approximately 3 hours after satellite overpass.
```
Comment (ZachTRice) [inline config/default/common/config/metadata/stories/fires_overview_2019/step001.md]: "Fire information available in Worldview are from" -> "Fire information available in Worldview **_is_** from"

File: config/default/common/config/metadata/stories/fires_overview_2019/step002.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<p>Data collected by the satellites utilize an algorithm that exploits the strong emission of mid-infrared radiation from fires and thermal anomalies - these fires and thermal anomalies are shown here as red points.  The points represent the center of a pixel within which one or multiple fires have occurred. For the MODIS instrument, the point represents the center of a 1km pixel; for the VIIRS instrument, the point represents the center of a 375m pixel. Using the eye icon, turn on and off the MODIS and VIIRS fire points to view the differences between the sources.
+</p>
+Find out more in the FIRMS FAQ:<br>
+[What does a MODIS active fire detection mean on the ground?](https://earthdata.nasa.gov/faq/firms-faq#ed-modis-fire-onground);<br>
```
Comment (ZachTRice) [inline config/default/common/config/metadata/stories/fires_overview_2019/step002.md]: Remove the semi-colon at the end of this line.

File: config/default/common/config/metadata/stories/fires_overview_2019/step006.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<p>The near real-time fire information are available through the Fire Information for Resource Management System (FIRMS). FIRMS provides the data for download in easy-to-use file formats, provides the ability for users to set up fire email alerts for their areas-of-interest and provides a fire map with additional information such as burned area.
+</p>
+Additional Information from FIRMS:<br>
+[Download Fire Data](https://firms.modaps.eosdis.nasa.gov/active_fire/#firms-shapefile)<br>
```
Comment (ZachTRice) [inline config/default/common/config/metadata/stories/fires_overview_2019/step006.md]: For list items add a "- " in front of them (unlinked)

Comment (minniewong) [reply config/default/common/config/metadata/stories/fires_overview_2019/step006.md]: Was trying to figure out how to do that!

File: config/default/common/config/metadata/stories/fires_overview_2019/step002.md (hunk 2 above)
Comment (minniewong) [reply config/default/common/config/metadata/stories/fires_overview_2019/step002.md]: done

File: config/default/common/config/metadata/stories/fires_overview_2019/step001.md (hunk 1 above)
Comment (minniewong) [reply config/default/common/config/metadata/stories/fires_overview_2019/step001.md]: reworded it

**Comment by ZachTRice** [comment]
@minniewong and I had a discussion about topic 2 - Step 2 the other day. It is not advisable to change the link color for one step because of usability / conformity (see: https://ux.stackexchange.com/questions/61101/should-all-links-on-a-website-be-the-same-color)
We experimented with changing the background (per article recommendation) on the in-progress modals to be a bit more consistent across all colors but ultimately decided that the current design looked best.

**Comment by ZachTRice** [comment]
Screenshot_3
Screenshot_2

**Comment by ZachTRice** [comment]
@rboller ^

**Comment by rboller** [comment]
I generally agree with keeping the color scheme consistent but unfortunately the legibility of the existing links is rather low IMO.  See attached.

I also agree that changing the background color could be a good option and also that the current color scheme is nice... except for viewing links.

So if changing the link color is off the table, I'd suggest tweaking the background color until a reasonable compromise is found between text legibility and overall good looks.  


fire-panel

**Comment by ZachTRice** [comment]
@rboller, @minniewong it appears I already had a "dark" modal style in place for the default dark-blue modals. I reused this style on the red and I believe it fixes the readability. I also updated the list style margin/padding. Please pull down the latest changes and have a look.

**Comment by minniewong** [comment]
@rboller and @ZachTRice - I made some edits to the text, please review the tour again!

**Pull request merged**